### PR TITLE
Add Rumahweb Icon

### LINKED ISSUE #8824: [request] rumahweb.com
### Brand Name

RumahWeb

### Website

rumahweb.com

### Popularity Metric

Global Rank
#102,563

https://www.similarweb.com/website/rumahweb.com/#overview

### Official Resources for Icon and Color

https://media.licdn.com/dms/image/C560BAQFEKIqs_TAmNQ/company-logo_200_200/0/1652062557883?e=2147483647&v=beta&t=FlWu9dWKVytVnzR1wtw_bXiNt0lr6AQDzJTNHAP1ee4

<details>
https://media.licdn.com/dms/image/C560BAQFEKIqs_TAmNQ/company-logo_200_200/0/1652062557883?e=2147483647&v=beta&t=FlWu9dWKVytVnzR1wtw_bXiNt0lr6AQDzJTNHAP1ee4
</details>

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Mvlprem**
rumahweb

**Issue:** closes #8824

**Similarweb rank:** [97,853](https://www.similarweb.com/website/rumahweb.com/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

svg taken from [website](https://www.rumahweb.com/) header, Hex taken from stylesheet ` --e-global-color-accent: ` ` #47CCFF `

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -12656,6 +12656,11 @@
             "hex": "FCC21B",
             "source": "https://github.com/charliermarsh/ruff/blob/e04ef423344837c916e75a7b09ea674711a104e0/assets/badge/v1.json"
         },
+        {
+            "title": "Rumahweb",
+            "hex": "47CCFF",
```
Comment (LitoMore) [inline _data/simple-icons.json:12661]:
```suggestion
            "hex": "2EB4E3",
```

Comment (LitoMore) [reply _data/simple-icons.json:12661]: The color should be `2EB4E3`. You can find the resource from the footer of the homepage.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (Mvlprem) [reply _data/simple-icons.json:12661]: Hello @LitoMore I assume you are talking about the colored logo in footer, but in pr #10253 @service-paradis said the hex is there to store the brand color, not the logo color.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
I think we should use the [logo](https://www.rumahweb.com/assets/img/svg/logo-rumahweb.svg) in the footer as the source here as it doesn't include the imperfections present in the one in the footer - would you mind updating the PR, please, @Mvlprem?

On the colour: my first instinct would have been to agree with your choice but, after seeing it referred to as an accent colour in their CSS variables, I think the shade used in that logo would be more accurate as the brand colour.

**Review by PeterShaggyNoble — APPROVED** [review]
Nice work, @Mvlprem 👍🏻

**Pull request merged**